server/utils/validation/gymEditSchema.ts

```diff
@@ -0,0 +1,22 @@
+import Joi from 'joi';
+
+const gymRegisterSchema = Joi.object({
+  gymName: Joi.string().required(),
+  logo: Joi.string().required(),
+  email: Joi.string()
+    .max(250)
+    .min(4)
+    .email({ minDomainSegments: 2, tlds: { allow: ['com', 'net'] } })
+    .required(),
+  password: Joi.string().max(25).min(8).required(),
```

Comment:
```suggestion

```
we will not send an email on edit profile, or password, and we will send gym ID in the request body